Update README.md

wey-chiang opened the pull request:
Added best practices, annotations, and fixed some nits

wey-chiang committed:
Update README.md

Added best practices, annotations, and fixed some nits

Signed-off-by: Weyland Chiang <71197790+wey-chiang@users.noreply.github.com>

villacaball commented on README.md:
@wey-chiang on line 7, could we explicitly note that SBOMs can be generated from manifest files please? So we'd want to add it to the comma separated list where we mention local filesystems and containers.

wey-chiang committed:
Update README.md

Explicit callout for package manifests and updated example for pointing to two files

Signed-off-by: Weyland Chiang <71197790+wey-chiang@users.noreply.github.com>
